xdump/__init__.py

```diff
@@ -1,4 +1,4 @@
 # coding: utf-8
 
 
-__version__ = "0.6.0"
+__version__ = "0.7.0"
```

Comment:
Please, revert this line. I usually make version bumps as separate commits when the release is ready.